backend/dal/Security/Permissions.cs

```diff
@@ -70,6 +70,18 @@ public enum Permissions
         LeaseEdit = 21,
 
         [Display(GroupName = "lease", Name = "lease-delete", Description = "Can delete existing leases.")]
-        LeaseDelete = 22
+        LeaseDelete = 22,
```

Comment:
hmm, thats odd. This doesn't seem to match up with the DB, let me check something.

Reply:
not relevant for this pr anyways I guess, will fix in mine re: permissions.